### PR TITLE
Set up build/deploy scripts for each service

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: github-event-service/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"use strict"
+// This shim file is in place to allow for deploying properly.
```
Comment (jelbourn) [inline github-event-service/index.js]: Can you expand this comment to explain this in more depth?

Comment (josephperrott) [reply github-event-service/index.js]: Oh yeah, I wrote this as a placeholder and then forgot to fill it out more.

File: github-event-service/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Object.defineProperty(exports, "__esModule", { value: true });
```
Comment (devversion) [inline github-event-service/index.js]: Why does this need the `__esModule` property? I'm pretty sure it runs without as well?

Comment (josephperrott) [reply github-event-service/index.js]: You're right, it doesn't.  Honestly, I just copy pasted that portion from the generated js files from a tsc build.

File: presubmit-service/package.json [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
-  "version": "0.0.1",
   "private": true,
-  "engines": {
-    "node": ">=8.0.0"
-  },
   "scripts": {
-    "start": "node ./index.js",
-    "gcp-build": "tsc index.ts"
+    "start": "node ./dist/index.js",
+    "build": "rm -rf dist/ && tsc -b tsconfig.json",
```
Comment (devversion) [inline presubmit-service/package.json]: What does `-b` do? I usually just do `tsc -p`

**Review by devversion — APPROVED** [review]
LGTM. Just the comment from Jeremy and the `tsc -b` thing I guess

File: presubmit-service/package.json (hunk 3 above)
Comment (josephperrott) [reply presubmit-service/package.json]: It builds the project and its dependencies if they are out of date, but `tsc -p tsconfig.json` is probably more appropriate.

**Pull request merged**